integration/helpers/node-template/package.json

```diff
@@ -18,7 +18,6 @@
     "isbot": "^4.1.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-router": "workspace:*",
```

Comment:
Shouldn't ever need this - everything should be re-exported via `react-router-dom`

Reply:
Yep, I meant to come back and clean this one up.